### PR TITLE
Phpstan upgrade

### PR DESCRIPTION
**Pull request by merk**

### CODE REVIEW

File: src/Bridge/Laravel/Providers/OrmServiceProvider.php [hunk 1]
```diff
@@ -21,15 +21,18 @@ public function register(): void
...
+        $this->app->singleton(
+            ResolveTargetEntityListener::class,
+            static function (Container $app): ResolveTargetEntityListener {
+                $listener = new ResolveTargetEntityListener();
 
-            $replacements = $app->make('config')->get('doctrine.replacements') ?? [];
-            foreach ($replacements as $abstract => $replacement) {
-                $listener->addResolveTargetEntity($abstract, $replacement, []);
-            }
+                $replacements = $app->make('config')->get('doctrine.replacements') ?? [];
```
Comment (edwardmurrell-loyaltycorp) [inline src/Bridge/Laravel/Providers/OrmServiceProvider.php]: Is this bit still required if we're no longer using our ORM wrapper?

File: src/Logger/Formatter/JsonFormatter.php [hunk 2]
```diff
@@ -9,7 +9,11 @@
 class JsonFormatter extends BaseJsonFormatter
 {
     /**
-     * {@inheritdoc}
+     * Formats a log record.
+     *
+     * @param mixed[] $record A record to format
+     *
+     * @return mixed The formatted record
```
Comment (edwardmurrell-loyaltycorp) [inline src/Logger/Formatter/JsonFormatter.php]: This look like it it'll be a string (or possibly null?)

File: src/Bridge/Laravel/Providers/OrmServiceProvider.php (hunk 1 above)
Comment (merk) [reply src/Bridge/Laravel/Providers/OrmServiceProvider.php]: I didnt make any changes to the provider except to add return typehints, evaluation of the actual code should be moved to other tickets

File: src/Logger/Formatter/JsonFormatter.php (hunk 2 above)
Comment (merk) [reply src/Logger/Formatter/JsonFormatter.php]: You're right, updated

File: src/Bridge/Laravel/Providers/OrmServiceProvider.php (hunk 1 above)
Comment (edwardmurrell-loyaltycorp) [reply src/Bridge/Laravel/Providers/OrmServiceProvider.php]: Fair, create a ticket?

**Pull request merged**